1209/F3FC/index.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: pid
+title: dRonin support for Lumenier Lux flight controller
```

Comment:
Thanks for your pull request! With this current title line, your entry will show up on the site as "dRonin dRonin support ...".

Can you provide a more summarised title, not including the orgname, that reflects what you'd expect to see if you plugged the device into a computer and looked at device manager or dmesg?